flowdb/sql/07_schema_cache.sql

```diff
@@ -37,4 +43,8 @@ CREATE TABLE IF NOT EXISTS cache.dependencies
                                     REFERENCES cache.cached (query_id) MATCH SIMPLE
                                     ON UPDATE NO ACTION
                                     ON DELETE CASCADE
-                            );
\ No newline at end of file
+                            );
+
+CREATE TABLE cache.cache_config (key text, value text);
+INSERT INTO cache.cache_config (key, value) VALUES ('half_life', 0);
```

Comment:
Worth initialising this with NULL just to be explicit that it'll be changed later?